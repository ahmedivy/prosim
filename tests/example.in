5 5 3
Proc1 3 1 1
DOOP 4
BLOCK 5
HALT

Proc2 3 1 1
BLOCK 4
DOOP 5
HALT

Proc3 3 1 2
DOOP 4
BLOCK 4
HALT

Proc4 3 1 2
DOOP 3
BLOCK 4
HALT

Proc4 3 1 3
DOOP 8
BLOCK 5
HALT
