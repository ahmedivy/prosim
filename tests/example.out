[01] 00000: process 1 new
[01] 00000: process 1 ready
[01] 00000: process 2 new
[01] 00000: process 2 blocked
[01] 00000: process 1 running
[01] 00004: process 2 ready
[01] 00004: process 1 blocked
[01] 00004: process 2 running
[03] 00000: process 1 new
[03] 00000: process 1 ready
[03] 00000: process 1 running
[03] 00005: process 1 ready
[03] 00005: process 1 running
[03] 00008: process 1 blocked
[03] 00013: process 1 finished
[02] 00000: process 1 new
[01] 00009: process 1 finished
[01] 00009: process 2 finished
[02] 00000: process 1 ready
[02] 00000: process 2 new
[02] 00000: process 2 ready
[02] 00000: process 1 running
[02] 00004: process 1 blocked
[02] 00004: process 2 running
[02] 00007: process 2 blocked
[02] 00008: process 1 finished
[02] 00011: process 2 finished
| 00008 | Proc 02.01 | Run 4, Block 4, Wait 0
| 00009 | Proc 01.01 | Run 4, Block 5, Wait 0
| 00009 | Proc 01.02 | Run 5, Block 4, Wait 0
| 00011 | Proc 02.02 | Run 3, Block 4, Wait 4
| 00013 | Proc 03.01 | Run 8, Block 5, Wait 0
